cscs-checks/tools/profiling_and_debugging/src/F90/gdb4hpc.slm

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -l
```

Comment:
@jgphpc, I think we should sync. Because the very existence of this file, means that either we need a new feature or that we should create this file by overwriting the init or setup methods.

I think this file should be created by the framework. We can leverage on the `emit_job` methods of the scheduler. IMHO, it should make the test more portable and more powerful (it could later be parameterised).

Reply:
both commits look ugly to me.

Reply:
If making changes to the framework is an option, i would ask for it. Otherwise, i'd be ok with sticking with the (dirty) solution for now.

Reply:
In order for gdb4hpc to work, you need 2 things:

1. decorate srun with:

```
#cray_debug_start
srun ./jacobi
#cray_debug_end
```

in gdb4hpc.slm, and 

2. a list of debug commands in gdb4hpc.in:

```
launch $tst{1} ./jacobi --sbatch=./gdb4hpc.slm
break _jacobi.F90:90
continue
list 90,91
print residual
...
```

Reply:
The problem lies in that the reframe job will launch another job

Reply:
@jgphpc I think that you can achieve what you want with the current functionality of ReFrame. We can have a look at it together.

Reply:
As agreed off-line, we should accept the test as is and discuss the feature that could improve the workflow later.